e2e/cypress/tests/integration/enterprise/cloud/billing/cloud_pricing_modal_spec.js

```diff
@@ -527,4 +527,28 @@ describe('Pricing modal', () => {
         cy.get('#pricingModal').get('#text-unselected').contains('Yearly');
         cy.get('#pricingModal').get('#professional').get('.plan_price_rate_section').contains(professionalMonthlySubscription.price_per_seat);
     });
+
+    it('Should display downgrade modal when downgrading from professional to free', () => {
+        const subscription = {
+            id: 'sub_test1',
+            product_id: 'prod_2',
+            is_free_trial: 'false',
+        };
+        simulateSubscription(subscription);
+        cy.apiLogout();
+        cy.apiAdminLogin();
+        cy.visit(urlL);
+
+        // # Open the pricing modal
+        cy.visit('/admin_console/billing/subscription?action=show_pricing_modal');
+
+        // * Pricing modal should be open
+        cy.get('#pricingModal').should('exist');
+
+        // * Click the free action (downgrade).
+        cy.get('#pricingModal').get('#free').get('#free_action').should('be.enabled').click();
```

Comment:
@ConorMacpherson a small suggestion,
we can separate this line into,
`cy.get(#pricingModal).should('be.visible')`
`cy.get(#free).should('be.visible')` ...

usually, `cy.get().get()` is avoided. I'm updating this in my PR too

Reply:
Ah, my bad. I've already merged this, but I can make this change still if you'd like.